packaging/windows/WINDOWS_INSTALLER.md

```diff
@@ -54,11 +54,17 @@ This section provides instructions for installing Netdata in silent mode, which
 Connect your Agent to your Netdata Cloud Space with token `<YOUR_TOKEN>` and room `<YOUR_ROOM>`:
 
 ```bash
-netdata-installer.exe /S /A /TOKEN=<YOUR_TOKEN> /ROOMS=<YOUR_ROOM>
+netdata-installer-x64.exe /S /A /TOKEN=<YOUR_TOKEN> /ROOMS=<YOUR_ROOM>
 ```
 
 Replace `<YOUR_TOKEN>` and `<YOUR_ROOM>` with your actual Netdata Cloud Space claim token and room ID, respectively.
 
+> **Note**
+>
+> The Windows version of Netdata is intended for users on paid plans.
+>
+> The Windows installer is currently under beta, and thus it is only available in the nightly release channel. A stable version will be released soon.
+
```

Comment:
I also added this snippet to be clear about the installer. Positioning was given by @netdata/product